openmetadata-ui/src/main/resources/ui/src/components/AlertBar/AlertBar.tsx

```diff
@@ -19,9 +19,13 @@ import { getIconAndClassName } from '../../utils/ToastUtils';
 import './alert-bar.style.less';
 import { AlertBarProps } from './AlertBar.interface';
 
-const AlertBar = ({ type, message }: AlertBarProps): JSX.Element => {
+const AlertBar = ({
+  type,
+  message,
+  isUnauthenticated,
+}: AlertBarProps): JSX.Element => {
   const { resetAlert, animationClass } = useAlertStore();
-  const [expanded, setExpanded] = useState(false);
+  const [expanded, setExpanded] = useState(isUnauthenticated);
```

Comment:
Instead of isUnauthenticated, it'd be propper to add a variable `defafultExpand` to make it generic